docs/community/governance.md

```diff
@@ -0,0 +1,106 @@
+# Governance
+
+## Hierarchy
+
+1. **Executive Heads**: The public face, managers and the decision makers.
+1. Rest -
+  1. **Core Team Members**: Freshers batch. (< 1 year of experience)
+  1. Executive Members: Batchmates of Executive Heads.
+  1. Advisors: People who have been an Executive member in the past and are still on the campus. They have seen a lot of KOSS, and the members should use their experience as a resource.
```

Comment:
The second line could be shifted into the description. Or it can be shortened to: "They advise Executives."